app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+    before_action :configure_permitted_parameters, if:  :devise_controller?
+
+
+
+
+    protected
```

Comment:
We can use private method here I think
```suggestion
    private
```

Reply:
Also, you can check rails styles guides for access modifiers indentation https://rubystyle.guide/#indent-public-private-protected